### DISCUSSION TITLE
Docs for V6?

### DISCUSSION THREAD

**Discussion by Gigamick** [katspaugh/wavesurfer.js#3569] [Q&A]
I have a production app using V6 and I need the docs. can't seem to find them on the site. I assume I'm being silly.

**Comment by katspaugh** [comment]
V6 readme: https://github.com/katspaugh/wavesurfer.js/tree/v6
V6 plugins docs: https://wavesurfer.xyz/plugins/
Old v6 examples: https://wavesurfer.xyz/example/
Archived API reference: https://web.archive.org/web/20220926034522/https://wavesurfer-js.org/docs/
